tox.ini

```diff
@@ -157,6 +157,35 @@ commands=
 commands_post=
     core: go tool covdata textfmt -i=.coverage -o coverage.txt
 
+# Add back mlflow in the mlflow PR
+[testenv:importer-wandb-py{38,39,310,311,312}]
+install_command=
+    pip install --timeout 600 --extra-index-url {opts} {packages}
+deps=
+    {[base]deps}
+    mlflow: pydantic<2
```

Comment:
why old pydantic? leave a comment on transitioning to the v2?

Reply:
`mlflow` requires pydantic<2